src/elements/content-uploader/__tests__/ItemRemove.test.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import noop from 'lodash/noop';
+
+import PlainButton from '../../../components/plain-button';
+import { ItemRemoveBase as ItemRemove } from '../ItemRemove';
+
+import { STATUS_ERROR, STATUS_IN_PROGRESS, STATUS_STAGED } from '../../../constants';
+
+describe('elements/content-uploader/ItemRemove', () => {
+    test.each`
```

Comment:
do we need this `test.each`? shouldn't the tooltip always appear regardless of the status?

Reply:
Hmm yeah, that's a good point. In that case shouldn't I test all possible statuses? Or just pick one?

Reply:
I would omit it altogether here since the status doesn't have an affect on the tooltip and their behaviors are independent

Reply:
if you wanted to test statuses, such as if `isDisabled` on `status === STATUS_STAGED`, then this would be a separate test

Reply:
I think the main difference between this component and the previous one (ItemAction) is that the tooltip text will change for the different statuses. but in this one, the text will always be "Remove"

Reply:
for the shallow wrapper, you could set any status. as we add more tests, then we could have a `getWrapper` implementation that would use the most common status but it still wouldn't affect the aria-label

Reply:
sounds good, will do!!